### PR TITLE
CEDS-3030 Fix random test failures due to misconstrued sample generation

### PR DESCRIPTION
**Pull request by udinobi**

### CODE REVIEW

**Comment by MaciejRewera** [comment]
Test this please.

File: app/models/FileUploadRequest.scala [hunk 1]
```diff
@@ -30,7 +30,7 @@ final case class FileUploadRequest(declarationId: MRN, files: Seq[FileUploadFile
     </FileUploadRequest>
 }
 
-sealed abstract case class FileUploadFile(fileSequenceNo: Int, documentType: String, serviceUrl: String) {
+sealed case class FileUploadFile(fileSequenceNo: Int, documentType: String, serviceUrl: String) {
```
Comment (MaciejRewera) [inline app/models/FileUploadRequest.scala:33]: Not sure about the purpose of `sealed` keyword here. `case class` should not be extended anyway.

File: test/generators/Generators.scala [hunk 2]
```diff
@@ -58,16 +49,21 @@ trait Generators extends ModelGenerators with JsonGenerators {
       choose(0, vector.size - 1).flatMap(vector(_))
     }
 
-  private val validEmailGen: Gen[String] = RegexpGen.from("[a-zA-Z0-9\\.!#$%&'*+/=?^_`{|}~-]{1,25}@[a-zA-Z0-9-.]{1,25}")
+  def eoriString: Gen[String] = alphaNumString(12)
```
Comment (MaciejRewera) [inline test/generators/Generators.scala:52]: This is actually incorrect. EORI has specific format so it should not be just alphanumeric.
According to the validation in Declarations service it is: 2 letters followed by 10 to 15 digits.

**Review by MaciejRewera — COMMENTED** [review]
Looks good. Left 2 little comments.

File: test/generators/Generators.scala (hunk 2 above)
Comment (udinobi) [reply test/generators/Generators.scala:52]: :+1:

**Review by MaciejRewera — APPROVED** [review]

**Pull request merged**